packages/components/src/Radio/Radio.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { Box, system } from '@marigold/system';
+import { Circle0, Circle1 } from '@marigold/icons';
+
+type RadioProps = {};
+
+export const Radio = system<RadioProps, 'input'>(
+  ({ variant = 'radio', checked, ...props }) => {
+    const [isCheckedOn, setChecked] = useState(false);
+    const toggleChecked = () => setChecked(!isCheckedOn);
+    return (
+      <Box>
+        <Box
+          as="input"
+          type="radio"
+          {...props}
+          css={{
+            position: 'absolute',
+            opacity: 0,
+            zIndex: -1,
+            width: 1,
+            height: 1,
+            overflow: 'hidden',
+          }}
+          onChange={() => toggleChecked()}
+          onClick={() => toggleChecked()}
```

Comment:
coverage is not on 100% because there are no tests for click events (as we talked about)